noxfile.py

```diff
@@ -9,7 +9,7 @@
 locations = "src", "tests"
 
 
-@nox.session(python=["3.7", "3.8", "3.9", "3.10"])
+@nox.session(python=["3.7", "3.8", "3.9", "3.10", "pypy-3.8", "pypy-3.9"])
```

Comment:
Let's drop 3.7?

Reply:
Yeah, are we doing it in this change or should we do follow up?

Reply:
Will do in a separate PR.